en/02_Developer_Guides/02_Controllers/02_Routing.md

```diff
@@ -303,6 +303,53 @@ class BreadAPIController extends Controller
 In Silverstripe CMS versions prior to 4.6, an empty key (`''`) must be used in place of the `'/'` key. When specifying an HTTP method, the empty string must be separated from the method (e.g. `'GET '`). The empty key and slash key are also equivalent in Director rules.
 [/alert]
 
+## Nested RequestHandlers
+
+Nested [`RequestHandler`](api:SilverStripe\Control\RequestHandler) routing is used extensively in the CMS and is used to create URL endpoints without yml configuration.  Nesting is done by returning a [`RequestHandler`](api:SilverStripe\Control\RequestHandler) from an action method on another [`RequestHandler`](api:SilverStripe\Control\RequestHandler), usually a [`Controller`](api:SilverStripe\Control\Controller).
+
+`RequestHandler` is the base class for other classes that can handle HTTP requests such as [`Controller`](api:SilverStripe\Control\Controller), [`FormRequestHandler`](api:SilverStripe\Forms\FormRequestHandler) (used by [`Form`](api:SilverStripe\Forms\Form)) and [`FormField`](api:SilverStripe\Forms\FormField).
+
+### How it works
+
+[`Director::handleRequest()`](api:SilverStripe\Control\Director::handleRequest()) begins the url parsing process by parsing the start of the URL and workng out which [`RequestHandler`](api:SilverStripe\Control\RequestHandler) subclass to use, looking in:
+
+- Routes set in yml config under Director `rules`
+- For `/admin` routes specifically, the `$url_segment` config for subclasses of `LeftAndMain`
```

Comment:
This makes it sound like `Director` is actually checking for the `$url_segment` in `LeftAndMain` subclasses.... but it's not. Director is just passing control of the request to `AdminRootController` because that's what `/admin` routes to (it's literally defined in `Director.rules` the same as all routes are), and then `AdminRootController` looks from there to find which `LeftAndMain` subclass (actually, which class returned from `CMSMenu::get_cms_classes()`) matches from there.

We don't need to go into all that much detail probably, but perhaps something like this would be more accurate:

```suggestion
- For `/admin` routes specifically, control of the request is passed to [`AdminRootController`](api:SilverStripe\Admin\AdminRootController)
```
Up to you if you then want to talk about the way this nests down to the `LeftAndMain` subclasses.

Reply:
Have removed reference to LeftAndMain and mentioned AdminRootController in the "Example of an AJAX `FormField` that uses nested `RequestHandler`'s" below